cli/jobs/deepspeed/deepspeed-autotuning/src/train.py

```diff
@@ -130,10 +131,7 @@ def add_argument():
 #
 # Showcasing logging metrics to automl.
 if args.with_aml_log:
```

Comment:
should we remove these 2 lines? we're not logging to AutoML per se.. and not sure we actually need to log something.

Reply:
We don't need to log here, it was just to show that you can log metrics from the train.py file. But we also have the loss metric being logged so I can remove it. What do you mean by we're not logging to AutoML per se?